src/tests/actions/articles/reportArticleActions.test.js

```diff
@@ -0,0 +1,91 @@
+import reduxThunk from 'redux-thunk';
+import configurestore from 'redux-mock-store';
+import * as moxios from 'moxios';
```

Comment:
Please remove the `*` from this import, and from all places where it doesn't need to be used.

Reply:
resolved